cardano-node/README.md

```diff
@@ -1,3 +1,103 @@
+<p align="center">
+  <big><strong>Cardano Node</strong></big>
+</p>
+
+![alt text](https://github.com/input-output-hk/cardano-node/blob/media/Cardano-ADA.png?raw=true)
+
+<p align="center">
+  <a href="https://github.com/input-output-hk/cardano-node/releases"><img src="https://img.shields.io/github/release-pre/input-output-hk/cardano-node.svg?style=for-the-badge" /></a>
+  <a href="https://buildkite.com/input-output-hk/cardano-node"><img src="https://img.shields.io/buildkite/a978cbb4def7018be3d0a004127da356f4db32f1c318c1a48a/master?label=BUILD&style=for-the-badge"/></a>
+</p>
+
+<hr/>
+
+- The cardano-node is the top level for the node and
+  aggregates the other components from other packages: consensus, ledger and
+  networking, with configuration, CLI, logging and monitoring.
+
+- The node no longer incorporates wallet or explorer functionality. The wallet
+  backend and explorer backend are separate components that run in separate
+  external processes that communicate with the node via local IPC.
+
+## How to build
+
+### Stack
+Use [Haskell Stack - Version 2.1.3](https://haskellstack.org/) to build this project:
+
+```
+$ cd cardano-node
+$ stack build
+```
+
+## Cabal
+
+Use [Cabal - Version 2.4.1.0](https://www.haskell.org/cabal/) to build this project:
+
+```
+$ cd cardano-node
+$ cabal new-build
```

Comment:
Perhaps we should recommend version 3.0? In which case it'd be just `cabal build`.

Reply:
Yes, good idea to also mention `cabal` v3.